openlibrary/catalog/marc/tests/test_data/xml_expect/warofrebellionco1473unit.json

```
@@ -27,8 +27,64 @@
   "publish_country": "dcu",
   "authors": [
     {
-      "name": "United States. War Dept.",
-      "entity_type": "org"
+      "birth_date": "1838",
+      "death_date": "1887",
+      "name": "Scott, Robert N.",
+      "entity_type": "person"
+    },
+    {
+      "birth_date": "1832",
+      "name": "Lazelle, Henry Martyn",
+      "entity_type": "person"
+    },
+    {
+      "birth_date": "1847",
+      "death_date": "1914",
+      "name": "Davis, George B.",
+      "entity_type": "person"
+    },
+    {
+      "name": "Perry, Leslie J.",
+      "entity_type": "person"
+    },
+    {
+      "birth_date": "1841",
+      "death_date": "1912",
+      "name": "Kirkley, Joseph W.",
+      "entity_type": "person"
+    },
+    {
+      "birth_date": "1852",
+      "death_date": "1834",
+      "name": "Ainsworth, Frederick Crayton",
+      "entity_type": "person"
+    },
+    {
+      "birth_date": "1842",
+      "name": "Moodey, John Sheldon",
+      "entity_type": "person"
+    },
+    {
+      "birth_date": "1849",
+      "name": "Cowles, Calvin D.",
+      "entity_type": "person",
+      "role": "comp"
```

```suggestion
      "role": "comp."
```
Keep trailing period.